app/views/spree/admin/orders/_shipment.html.haml

```diff
@@ -85,3 +85,25 @@
         %td.actions
           - if can?(:update, shipment) && shipment.can_modify?
             = link_to '', '', :class => 'edit-tracking icon_link icon-edit no-text with-tip', :data => { :action => 'edit' }, :title => Spree.t('edit')
+      
+      %tr.edit-note.hidden.total
+        %td{ :colspan => "5" }
+          %label
+            = "#{Spree.t(:note)}:"
```

Comment:
Our convention for new translations is to scope them to the view file:
```suggestion
            = t(".note_label")
```

I would then include the `:` in the translation as well. Different languages may use different symbols for the meaning of "this label applies to the following content".

Reply:
That was a copy/paste from the _tracking number_ just above, but yeah, let's correct it! 🙏